index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sameed - Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+</head>
+
+
+
+<body background="./Images for portfolio/header-shapes mobile@2x.svg">
```

Comment:
- [ ] As mentioned by the reviewers before me, please remove this image from the HTML and add it passing by CSS instead, this will make your styling more flexible and contribute to your dry, clean code and professionalism
- [ ] Concerning the duplicated background, I think you need to use the `background-repeat: no-repeat` CSS property to solve it, but because you are putting the same bg image for your entire app(body), this should create some issues next time, what you can do is, give a class to the section on line 30 and use your CSS bg properties to set it there., hop this helps @sameedmohsin28.
- [ ] [ OPTIONAL ] I just mark this one optional but it should not be so, not much to say here, I just call your attention to details, see the 2 images and make the necessary to match as much as possible the design on the left.

<img width="760" alt="1" src="https://user-images.githubusercontent.com/77843050/212502982-8375cecf-48c8-433f-97d8-681c4a8110e7.png">